web/src/lib/components/shared-components/settings/setting-accordion.svelte

```diff
@@ -38,20 +40,28 @@
   });
 </script>
 
-<div class="border-b-[1px] border-gray-200 py-4 dark:border-gray-700" bind:this={accordionElement}>
+<div
+  class={`border rounded-2xl my-4 px-6 py-4 transition-all ${isOpen ? 'border-immich-primary/40 dark:border-immich-dark-primary/50 shadow-md' : 'dark:border-gray-800'}`}
+  bind:this={accordionElement}
+>
   <button
     type="button"
     aria-expanded={isOpen}
     on:click={() => (isOpen = !isOpen)}
     class="flex w-full place-items-center justify-between text-left"
   >
     <div>
-      <h2 class="font-medium text-immich-primary dark:text-immich-dark-primary">
-        {title}
-      </h2>
+      <div class="flex gap-2 place-items-center">
+        {#if icon}
+          <Icon path={icon} class="text-immich-primary dark:text-immich-dark-primary" size="24" />
```

Comment:
Small change - hiding this icon from the screen reader.

```suggestion
          <Icon path={icon} class="text-immich-primary dark:text-immich-dark-primary" size="24" ariaHidden />
```